### PR TITLE
Add link to Twitter

### PR DESCRIPTION
**Pull request by ozkriff**
This PR adds a social link to https://twitter.com/rust_gamedev.

### CODE REVIEW

File: _includes/social.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
...
   {%- for mst in social.mastodon -%}{%- if mst.username and mst.instance -%}<li><a rel="me" href="https://{{ mst.instance | cgi_escape | escape}}/@{{mst.username}}" title="{{ mst.username | escape }}"><svg class="svg-icon grey"><use xlink:href="{{ '/assets/minima-social-icons.svg#mastodon' | relative_url }}"></use></svg></a></li>{%- endif -%}{%- endfor -%}
-  {%- if social.twitter -%}<li><a href="https://twitter.com/{{ social.twitter | cgi_escape | escape }}" title="{{ social.twitter | escape }}"><svg class="svg-icon grey"><use xlink:href="{{ '/assets/minima-social-icons.svg#twitter' | relative_url }}"></use></svg></a></li>{%- endif -%}
+  {%- if social.twitter -%}<li><a href="https://twitter.com/{{ social.twitter | cgi_escape | escape }}" title="{{ social.twitter | escape }}"><svg class="svg-icon grey"><use xlink:href="{{ '/assets/minima-social-icons.svg#twitter' | relative_url }}"></use></svg> Twitter (rust_gamedev)</a></li>{%- endif -%}
```
Comment (ozkriff) [inline _includes/social.html]: This change was made to match the style of Github and Discord links

Comment (AlexEne) [reply _includes/social.html]: Who owns that twitter?

Comment (ozkriff) [reply _includes/social.html]: Me. Also, @erlend-sh is on the (tweetdeck) team:

image

Comment (AlexEne) [reply _includes/social.html]: Ahh nice

**Comment by ozkriff** [comment]
So, of there's no objection, I'll merge this PR today.

**Pull request merged**